arches_references/management/commands/controlled_lists.py

```diff
@@ -218,60 +249,51 @@ def migrate_concept_nodes_to_reference_datatype(self, graph_id):
                     node.full_clean()
                     node.save()
 
-                    cross_records = (
-                        node.cardxnodexwidget_set.annotate(
-                            config_without_i18n=Cast(
-                                models.F("config"),
-                                output_field=models.JSONField(),
-                            )
+                    cross_records = node.cardxnodexwidget_set.annotate(
+                        config_without_i18n=Cast(
```

Comment:
This annotation being called "config_without_i18n" is a little confusing, since we don't seem to be minus'ing the i18n anywhere. Can we delete this and just use the single annotate() call below (where the left hand side is just F("config")?